lib/wallet/src/Cardano/Wallet/Transaction/Voting.hs

```diff
@@ -0,0 +1,122 @@
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE LambdaCase #-}
+
+-- |
+-- Copyright: © 2024 Cardano Foundation
+-- License: Apache-2.0
+--
+-- Tools for creating transactions that change the voting status.
+--
+module Cardano.Wallet.Transaction.Voting
+    ( certificateFromVotingAction
+    ) where
+
+import Prelude
+
+import Cardano.Address.Derivation
+    ( XPub
+    , xpubPublicKey
+    )
+import Cardano.Address.Script
+    ( KeyHash (..)
+    , Script (..)
+    )
+import Cardano.Crypto.Hash.Class
+    ( Hash (UnsafeHash)
+    )
+import Cardano.Wallet.Primitive.Ledger.Convert
+    ( toLedgerDelegatee
+    )
+import Cardano.Wallet.Primitive.Ledger.Shelley
+    ( toCardanoLovelace
+    , toCardanoSimpleScript
+    )
+import Cardano.Wallet.Primitive.Types.Coin
+    ( Coin
+    )
+import Cardano.Wallet.Transaction
+    ( VotingAction (..)
+    )
+import Crypto.Hash.Extra
+    ( blake2b224
+    )
+import Data.ByteString.Short
+    ( toShort
+    )
+
+import qualified Cardano.Api as Cardano
+import qualified Cardano.Api.ReexposeLedger as Ledger
+import qualified Cardano.Api.Shelley as Cardano
+import qualified Internal.Cardano.Write.Tx as Write
+    ( CardanoApiEra
+    , RecentEra (RecentEraBabbage, RecentEraConway)
+    )
```

Comment:
```suggestion
```

(🤷 I see no value in adding explicit exports to qualified imports.)

Reply:
yep, the same was in Transaction.Delegation module and I automatically copy pasted it. Now removed that strange way of importing in both modules